runtime/include/tasks/qthreads/tasks-qthreads.h

```diff
@@ -134,13 +134,6 @@ typedef struct {
           .requestedSubloc = _subloc, \
           .prvdata = { .serial_state = _serial } }
 
-typedef struct {
-    void                     *fn;
-    void                     *args;
-    chpl_bool                countRunning;
-    chpl_task_prvDataImpl_t  chpl_data;
-} chpl_qthread_wrapper_args_t;
-
```

Comment:
Right, apparently at some point we no longer needed it out in the .h, so here I'm just doing information-hiding.